src/utils.ts

```diff
@@ -33,6 +33,23 @@ import tokenCache from "./tokens.json";
 export const NATIVE_CONTRACT_ADDRESS =
   "0x0000000000000000000000000000000000000000";
 
+const defaultDeadlines: {
```

Comment:
can you just add a comment above this explaining these values come from https://sia.mayan.finance/v3/init ?